lms/djangoapps/verify_student/models.py

```diff
@@ -351,11 +351,22 @@ class PhotoVerification(IDVerificationAttempt):
     # capturing it so that we can later query for the common problems.
     error_code = models.CharField(blank=True, max_length=50)
 
+    # Mark if attempt is going through an A/B experiment
+    experiment_name = models.CharField(null=True, max_length=50)
```

Comment:
I assume this would be a temporary field on the model and that it would be removed after the need for experimentation is over. Can you file a ticket for the removal of this field and reference it here?